core/src/OpenMP/Kokkos_OpenMP_Instance.hpp

```diff
@@ -135,15 +135,15 @@ namespace Impl {
 // Partitioning an Execution Space: expects space and integer arguments for
 // relative weight
 template <typename T>
-inline std::vector<OpenMP> create_OpenMP_instances(
+inline std::vector<int> calculate_omp_pool_sizes(
```

Comment:
Changed this to just calculate pool sizes, instead of returning `OpenMP` instances since `partition_space` expects an array and a vector version of the instances.